Neos.Flow.Log/Tests/Unit/LoggerTest.php

```diff
@@ -28,7 +28,7 @@ public function logPassesItsArgumentsToTheBackendsAppendMethod()
         $mockBackend = $this->getMockBuilder(BackendInterface::class)->setMethods(['open', 'append', 'close'])->getMock();
         $mockBackend->expects($this->once())->method('append')->with('theMessage', 2, ['foo'], 'Foo', 'Bar', 'Baz');
 
-        $logger = new Logger();
+        $logger = new DefaultLogger();
```

Comment:
Why do you use deprecated class here instead of `Psr\Logger`? For the ease of migration?

Reply:
These are the logger unit tests, so IMHO they should just continue testing the original logger even if it's deprecated.

Reply:
Oh doh, sorry, didn't see it was a test